compiler/circle-opselector-test/README.md

```diff
@@ -0,0 +1,52 @@
+# Circle-OpSelector-test
+
+## Usage
+
+- This test didn't consider **if** and **while** nodes yet. 
+
+**Single file test**
+
+- `tflite-file-name` must be contained in [tflite receipes](https://github.com/Samsung/ONE/tree/master/res/TensorFlowLiteRecipes)
+- `selected nodes` format is `"0 1 3"`
+
+```
+$ ./opselector_test.sh [tflite-file-name] [selected nodes]
+# ex)
+$ ./compiler/circle-opselector-test/opselector_test.sh Part_Sqrt_Rsqrt_Add_002 "0 1 2 3"
+```
+
+This test compares between the result of `circle-opselector` and [select-operator.py](https://github.com/Samsung/ONE/blob/master/tools/tflitefile_tool/select_operator.py) in tflite_tool.
+
+The detailed process is as follows
+
+1. Prepare a `tflite` model.
+2. Create a `circle` version of that `tflite` model using 'tflite2circle`.
+3. Create a `tflite` subgraph using `select_operator.py`.
+4. Create a `circle` subgraph using `Circle-OpSelector`.
+5. Run `nnpckgtc` with input as the `tflite` subgraph to get the golden value and the package.
+6. Modify the **package** so that it contains the `circle` subgraph, not the `tflite` subgraph. This step is due to the fact that the circle model can't be an input of `nnpckgtc`.
+7. Run the **package** in **runtime** to get the output of the `circle` subgraph package, with the same input used for getting the golden value.
```

Comment:
Just curious, to run with `runtime` are you planning to run this in ARM device?
or do we have all working kernels in x86-64?

Reply:
We had all working kernels in x86-64. So we are not convinced that run `runtime` on ARM device. But I think it doesn't need to test on ARM device because circle-opselector just would be used on x86-64.

Reply:
OK, use x86-64 runtime...

This will require to test ONE frontend, (1) we need to build x86-64 runtime, which does not exist as of now and I don't think this is a good idea, which will take more time. or (2) install runtime debian package in the test process
Either way, we need to change test environment althought we already have `luci-interpreter` and `tflite` python envionment for testing.

So, I do not agree to land thest changes.

Reply:
@parkeunsang Please follow `compiler`'s way (e.g. `luci-interpreter` or anything recommanded by compiler part guys). Though I believe `onert` is easier to use (download prebuilt release binaries and run existing script) and has more operator coverage (not true for your case `x86-64`), it is recommended at early development stage in your local development environment, it is not for integrating with existing `compiler`s as @seanshpark commented. For how-to, see comments (including #7606).

Reply:
@parkeunsang FYI, there are several reasons why the module that uses runtime modules can't be included in `compiler` directory.

As you know, runtime and compiler have separate build system and all compiler modules are in the `compiler` folder. Accordingly, we have separate CI system as well. So, compiler test modules have to use the module located in the same `compiler` directory. Plus, release process and packages are all different, either.

So, you have two options.

### Go as it is

If you want to stick with the current method, please move these test codes to `tools` directory in root folder. This folder has several scripts that are not included in build system.

### Use `luci-interpreter`

You can see #7606 to achieve this. And, if you follow this option, you should write `CMakeLists.txt` to include the test on compiler build system and ctest.

Reply:
I will use `luci-interpreter` thanks